### ISSUE TITLE
ix-blind header-actions slot layout broken in iX 2.2

### ISSUE THREAD

**Issue by Domi1980** [siemens/ix#1279]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

Since upgrade to iX 2.2.x, the layout of slot header-actions of ix-blind is broken. The content of the slot is now multline in our scenario, which contains an ix-icon and an input type checkbox.
With iX 2.1.2 the content was on a single line without line-wrap.

This seems to have been caused by https://github.com/siemens/ix/pull/1186

### What type of frontend framework are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.2.1

### Code to produce this issue.

```shell
example for iX 2.1.2 (working): https://stackblitz.com/edit/ylneor?file=src%2Fapp%2Fblind-header-actions.ts
example with 2.2.1 (broken): https://stackblitz.com/edit/ylneor-9kru5c?file=src%2Fapp%2Fblind-header-actions.ts
```

**Comment by danielleroux** [comment]
Fix will be shipped with next release [v2.3.0](https://github.com/siemens/ix/pull/1257)

**Comment by SusiHorter** [comment]
@danielleroux  is there an ETA for v2.3.0?
